testnet/stacks-node/Cargo.toml

```diff
@@ -33,15 +40,27 @@ libsigner = { path = "../../libsigner" }
 ring = "0.16.19"
 warp = "0.3.5"
 tokio = "1.15"
-reqwest = { version = "0.11", default_features = false, features = ["blocking", "json", "rustls", "rustls-tls"] }
-clarity = { path = "../../clarity", features = ["default", "testing"]}
-stacks-common = { path = "../../stacks-common", features = ["default", "testing"] }
-stacks = { package = "stackslib", path = "../../stackslib", features = ["default", "testing"] }
-stacks-signer = { path = "../../stacks-signer" }
+reqwest = { version = "0.11", default_features = false, features = [
```

Comment:
Generally not a fan of shipping a reformatting of unrelated files with a feature PR, since it distracts the reviewer (like, I have to go and manually check that all of the settings are the same before and after the reformat).  Can you just revert this for now?